sdk/src/client/error.rs

```diff
@@ -86,25 +86,22 @@ pub enum Error {
     MissingParameter(&'static str),
     /// Error on API request
     #[error("node error: {0}")]
-    Node(String),
+    Node(#[from] crate::client::node_api::error::Error),
     /// The block doesn't need to be promoted or reattached
     #[error("block ID `{0}` doesn't need to be promoted or reattached")]
     NoNeedPromoteOrReattach(String),
     /// The requested data was not found.
     #[error("the requested data {0} was not found.")]
     NotFound(String),
```

Comment:
Yes it's still used. Not found is when a quorum question has no result. The other one, noresult is per node basis

Reply:
I'm not speaking of any `noresult`. We have both `ClientError::NotFound` and `NodeError::NotFound` but in `node_api` module we use the client one where I'd have expected a node error to be used? And why wouldn't node quorum error be in node error? Maybe they need to be renamed for a bit of clarity, it's a bit confusing.

Reply:
IM open to suggestions:
The node `NotFound` gets thrown when the page returns 404
The client `NotFound` gets returned when on a call to multiple nodes, no result is found ever. However i do think we can unwrap there because that means the `self.http_client.get_bytes` doesnt return anything ever, no ok or err. Lets remove it and ill see if i can get rid of NotFound in client::Error

Reply:
Just curious: why do we prefer unwrapping than just using the other NotFound instead?

Reply:
So its not confusing which one means what, rather just have it gone completely in this case i think

Reply:
Hmm ok, as long as you are 200% sure it can't panic